tests/test_au/tests.py

```diff
@@ -106,3 +118,86 @@ def test_AUPhoneNumberField(self):
             '1800DJANGO': error_format,
         }
         self.assertFieldOutput(AUPhoneNumberField, valid, invalid)
+
+    def test_abn(self):
+        error_format = ['Enter a valid ABN.']
+        valid = {
+            '53004085616': '53004085616',
+            '53 004 085 616': '53004085616',
+        }
+        invalid = {
+            '53004085617': error_format,
+            '5300A085616': error_format,
+        }
+        self.assertFieldOutput(AUBusinessNumberField, valid, invalid)
+
+
+class AULocalFlavorAUBusinessNumberFieldValidatorTests(TestCase):
+
+    def test_no_error_for_a_valid_abn(self):
+        """Test a valid ABN does not cause an error."""
+        valid_abn = '53004085616'
+
+        validator = AUBusinessNumberFieldValidator()
+
+        try:
+            validator(valid_abn)
+        except ValidationError:
+            self.fail('Validation unexpectedly failed.')
```

Comment:
You don't need the try / except. You can just run the validator. If it fails, the tests will break which is what we want.